src/csvcubed/schema/codelist-config/v1_2/schema.json

```diff
@@ -0,0 +1,182 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema",
+    "id": "https://purl.org/csv-cubed/code-list-config/v1.1",
```

Comment:
I might be wrong but shouldn't this be v1.1?

Reply:
Good spot, updated